### PR TITLE
Update the WP packages with fixes prior to WP 6.2 RC2

### PR DESCRIPTION
**Pull request by Mamaduka**
Trac ticket: https://core.trac.wordpress.org/ticket/57471

This updated the packages to the latest versions with the latest fixes.

https://github.com/WordPress/gutenberg/commit/f22a3cbf0cd8f48c7ef800a09b69212a893b99b8

Includes the following changes:

- ~~[Bug] Comments: Fix 'sprintf requires more than 1 params' error https://github.com/WordPress/gutenberg/pull/49054 > Introduced in 6.1.1 (not in 6.2)~~ ❌ 
- [Regression] Don't offer Classic block as a recovery action when not registered https://github.com/WordPress/gutenberg/pull/49051 ✅ 
- [i18n Change] Fix typo in the media-categories component https://github.com/WordPress/gutenberg/pull/49047
- Hide navigation screen in site editor https://github.com/WordPress/gutenberg/pull/49043 ✅ > (Feature is not ready for 6.2 - see Make/Core post for details)[https://make.wordpress.org/core/2023/03/14/fyi-navigation-section-of-new-site-editor-experienced-removed-for-6-2-rc-2/]
- [i18n Change] Site editor: Fix non-us spelling in sidebar https://github.com/WordPress/gutenberg/pull/48976 ✅ > Agreed to fix this in RC
- Site Editor: Fix lingering insertion point within template parts https://github.com/WordPress/gutenberg/pull/48913 ✅ > Regression was introduced in 6.2 cycle
- Navigation Link: Remove color generation code https://github.com/WordPress/gutenberg/pull/48927 and https://github.com/WordPress/gutenberg/pull/49064 ✅
- Fix settings tab active state border in block inspector https://github.com/WordPress/gutenberg/pull/48945 ✅ > This bug may have been introduced in the 6.2 cycle.
- Fix text alignment in the Site Editor sidebar  https://github.com/WordPress/gutenberg/pull/48959 ✅ Making template descriptions more prominent is part of 6.2, not unusure if this alignment issue was introduced in 6.2 or not.

### CODE REVIEW

File: src/wp-includes/blocks/comment-template.php [hunk 1]
```diff
@@ -58,11 +58,10 @@ function block_core_comment_template_render_comments( $comments, $block ) {
 				$block_content .= sprintf( '<ol>%1$s</ol>', $inner_content );
 				--$comment_depth;
 			} else {
-				$inner_content  = block_core_comment_template_render_comments(
+				$block_content .= block_core_comment_template_render_comments(
 					$children,
 					$block
 				);
-				$block_content .= sprintf( $inner_content );
```
Comment (Mamaduka) [inline src/wp-includes/blocks/comment-template.php]: See: https://github.com/WordPress/gutenberg/pull/49054

File: src/wp-includes/blocks/navigation-link.php [hunk 2]
```diff
@@ -5,76 +5,6 @@
...
-/**
- * Build an array with CSS classes and inline styles defining the colors
- * which will be applied to the navigation markup in the front-end.
- *
- * @param  array $context     Navigation block context.
- * @param  array $attributes  Block attributes.
- * @param  bool  $is_sub_menu Whether the link is part of a sub-menu.
- * @return array Colors CSS classes and inline styles.
- */
-function block_core_navigation_link_build_css_colors( $context, $attributes, $is_sub_menu = false ) {
```
Comment (Mamaduka) [inline src/wp-includes/blocks/navigation-link.php]: See: https://github.com/WordPress/gutenberg/pull/48927

**Comment by audrasjb** [comment]
Alright, looks good to me.
Note for the Core commit: this also fixes Trac ticket 57883.

**Review by audrasjb — APPROVED** [review]

**Comment by hellofromtonya** [comment]
Currently reviewing. There is at least 1 fix (so far) that was a bug introduced in 6.1, not in 6.2.

**Review by hellofromtonya — CHANGES_REQUESTED** [review]
Blocking commit until the review of the fixes is completed. Why? To ensure all fixes are bugs and/or regressions introduced in the 6.2 cycle. There is at least 1 so far from 6.1.

**Comment by hellofromtonya** [comment]
@Mamaduka I've reviewed each of the Gutenberg PRs to determine if each bug / regression was introduced in the 6.2 cycle. 

This bug was not introduced in 6.2:

- [Bug] Comments: Fix 'sprintf requires more than 1 params' error https://github.com/WordPress/gutenberg/pull/49054 > Introduced in 6.1.1 (not in 6.2) ❌ 

This fix is removing a PHP function. Was this function introduced in 6.2?
- Navigation Link: Remove color generation code https://github.com/WordPress/gutenberg/pull/48927 ❓ > Bug was introduced in 6.2 Beta cycle, but not in RC. And a PHP function is being removed. Need to investigate when this function was introduced into Core.

I also left comments on each of these PRs.

**Comment by Mamaduka** [comment]
@hellofromtonya,

1. **Comments: Fix 'sprintf requires more than 1 params' error** - Don't have a strong opinion here. I would love to fix the fatal error, but if it's out of the scope of the RCs, then happy to revert the change.
2. **Navigation Link: Remove color generation code** - We can also fix this issue without removing the function and applying changes from this area - https://github.com/WordPress/wordpress-develop/pull/4234/files#diff-e3c55a4e55bda8c2b88525cb3627924cefcfd1966f936ebb3370020eb296be94R110.

**Comment by hellofromtonya** [comment]
>Navigation Link: Remove color generation code - We can also fix this issue without removing the function and applying changes from this area - https://github.com/WordPress/wordpress-develop/pull/4234/files#diff-e3c55a4e55bda8c2b88525cb3627924cefcfd1966f936ebb3370020eb296be94R110.

@Mamaduka Do you when that function was introduced into Core? If during 6.2, then removing is fine. Else, could be deprecated instead if not used and the alternative fix applied.

**Comment by hellofromtonya** [comment]
>Comments: Fix 'sprintf requires more than 1 params' error - Don't have a strong opinion here. I would love to fix the fatal error, but if it's out of the scope of the RCs, then happy to revert the change.

This one has a Trac ticket open for it https://core.trac.wordpress.org/ticket/57883. Let's revert it from this PR and continue discussions in that Trac ticket. Sound good? @Mamaduka

**Comment by Mamaduka** [comment]
@hellofromtonya, definitely not in 6.2. Here is the PR that introduced the function into the Gutenberg plugin - https://github.com/WordPress/gutenberg/pull/21075/, so probably WP 5.5-5.6

**Comment by hellofromtonya** [comment]
>definitely not in 6.2. Here is the PR that introduced the function into the Gutenberg plugin - https://github.com/WordPress/gutenberg/pull/21075, so probably WP 5.5-5.6

Okay thank you @Mamaduka. A hard removal of the function breaks Core's BC (Backwards Compatibility) Promise. It can be deprecated instead or fixed. See my comment here https://github.com/WordPress/gutenberg/pull/48927#issuecomment-1468117884

**Comment by hellofromtonya** [comment]
>This one has a Trac ticket open for it https://core.trac.wordpress.org/ticket/57883. Let's revert it from this PR and continue discussions in that Trac ticket.

Sorry for misguidance here. @Mamaduka Let's leave this fix on this PR for now but make it a separate commit please. That way its commit can be excluded if there's a decision not to commit it.

**Comment by Mamaduka** [comment]
- **Comments: Fix 'sprintf requires more than 1 params' error** - Reverted in https://github.com/WordPress/gutenberg/commit/4486a6b22efb62365e2e5976263883558716da8c.
- **Navigation Link: Remove color generation code** - Feedback addressed in https://github.com/WordPress/gutenberg/pull/49064.

**Comment by Mamaduka** [comment]
> Let's leave this fix on this PR for now but make it a separate commit please. That way its commit can be excluded if there's a decision not to commit it.

@hellofromtonya, it needs to be in a separate package update; otherwise build process will overwrite the changes.

Can we include it in RC3 when the decision is reached?

**Comment by hellofromtonya** [comment]
Published package updates https://github.com/WordPress/gutenberg/commit/356298f91f5e43d68dda6a770e3e6f8e1aa69f14

**Review by hellofromtonya — APPROVED** [review]
* Confirmed the package version numbers match the 2 publish releases https://github.com/WordPress/gutenberg/commit/f22a3cbf0cd8f48c7ef800a09b69212a893b99b8 and https://github.com/WordPress/gutenberg/commit/356298f91f5e43d68dda6a770e3e6f8e1aa69f14  ✅ 
* Confirmed the removed PHP function is restored to avoid breaking BC ✅ 
* Confirmed the `sprintf()` bug introduced in 6.1.0 is not in this package update as it is still being discussed ✅ 

Ready for commit 👍

**Comment by hellofromtonya** [comment]
Committed via https://core.trac.wordpress.org/changeset/55542 into `trunk`.

**Pull request closed without merging**